Mathlib/Data/PFunctor/Multivariate/W.lean

```diff
@@ -57,63 +57,63 @@ open MvFunctor
 variable {n : ℕ} (P : MvPFunctor.{u} (n + 1))
 
 /-- A path from the root of a tree to one of its node -/
-inductive WPath : P.last.W → Fin2 n → Type u
-  | root (a : P.A) (f : P.last.B a → P.last.W) (i : Fin2 n) (c : P.drop.B a i) : WPath ⟨a, f⟩ i
+inductive wpath : P.last.W → Fin2 n → Type u
```

Comment:
This looks wrong, types are capitalized.

Reply:
Sorry, I mistook.